hack/manifests/backstage/Readme.md

```diff
@@ -0,0 +1,56 @@
+# Install parados on plain kubernetes
+
+
+## Install kind
```

Comment:
this title is confusing: I expected steps to install `kind`  (or a link to https://kind.sigs.k8s.io/docs/user/quick-start/) but instead, it is the command to initialize the local cluster with kind